source/jormungandr/jormungandr/instance.py

```diff
@@ -185,11 +186,23 @@ def __init__(
 
         self.zmq_socket_type = zmq_socket_type
 
-        self.equipment_providers_ids = instance_equipment_providers
-        self.equipment_provider_manager = equipment_provider_manager
+        if app.config['DISABLE_DATABASE']:
+            self.equipment_provider_manager = EquipmentProviderManager(
+                app.config[str('EQUIPMENT_DETAILS_PROVIDERS')]
+            )
+        else:
+            self.equipment_provider_manager = EquipmentProviderManager(
+                app.config[str('EQUIPMENT_DETAILS_PROVIDERS')], self.get_providers_from_db
+            )
 
-        # Create only equipment providers defined in the instance
-        self.equipment_provider_manager.init_providers(self.equipment_providers_ids)
+        # Init equipment providers from config
+        self.equipment_provider_manager.init_providers(instance_equipment_providers)
+
+    def get_providers_from_db(self):
```

Comment:
does calling this fuction trigger some sql queries, except from the one from `get_models`. If it is the case we will want to cache them.
with `lazy=joined` it should not, but better be safe than sorry

Reply:
Tested. No other request made!